lib/src/tileset.dart

```diff
@@ -8,11 +9,17 @@ import 'themes/theme_layers.dart';
 class Tileset {
   final bool preprocessed;
   final Map<String, VectorTile> tiles;
+  late final ThemeLayerFeatureResolver themeLayerFeatureResolver;
```

Comment:
for previty consider renaming this field to `layerFeatureResolver` or just `resolver`